src/init.cpp

```diff
@@ -1212,6 +1236,14 @@ bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
     }
 #endif
 
+#if ENABLE_PROTON
+    pAMQPNotificationInterface = AMQPNotificationInterface::CreateWithArguments(mapArgs);
+
+    if (pAMQPNotificationInterface) {
```

Comment:
Same as the ZMQ code block above.  Note that ::CreateWithArguments does log failure to initialize a notifier: https://github.com/bitcartel/zcash/blob/daa4d27e6815a144ca5f60cdf633f9732f72153f/src/amqp/amqpnotificationinterface.cpp#L98